internal/page/donor/identity_with_one_login_callback.go

```diff
@@ -63,6 +64,22 @@ func IdentityWithOneLoginCallback(tmpl template.Template, oneLoginClient OneLogi
 			return err
 		}
 
+		if len(userInfo.ReturnCodes) > 0 {
+			for _, c := range userInfo.ReturnCodes {
+				if c.Code == "X" {
```

Comment:
I think you could clean this up by having the logic in `onelogin.Client.ParseIdentityClaim(...)` as that returns a `identity.UserData` already